include/zephyr/posix/signal.h

```diff
@@ -12,6 +12,62 @@
 extern "C" {
 #endif
 
+#ifdef CONFIG_POSIX_SIGNAL
+enum {
+	SIGABRT = 1,
+	SIGALRM,
+	SIGBUS,
+	SIGCHLD,
+	SIGCONT,
+	SIGFPE,
+	SIGHUP,
+	SIGILL,
+	SIGINT,
+	SIGKILL,
+	SIGPIPE,
+	SIGQUIT,
+	SIGSEGV,
+	SIGSTOP,
+	SIGTERM,
+	SIGTSTP,
+	SIGTTIN,
+	SIGTTOU,
+	SIGUSR1,
+	SIGUSR2,
+	SIGPOLL,
+	SIGPROF,
+	SIGSYS,
+	SIGTRAP,
+	SIGURG,
+	SIGVTALRM,
+	SIGXCPU,
+	SIGXFSZ,
+};
+
+/* Signal set management definitions and macros. */
+#define MIN_SIGNO       1                       /* Lowest valid signal number */
+#define MAX_SIGNO       CONFIG_POSIX_SIGNAL_MAX /* Highest valid signal number */
+
+/* Definitions for "standard" signals */
+#define SIGSTDMIN       1                           /* First standard signal number */
+#define SIGSTDMAX       CONFIG_POSIX_SIGNAL_STD_MAX /* Last standard signal number */
+
+/* Definitions for "real time" signals */
+#define SIGRTMIN        (SIGSTDMAX + 1) /* First real time signal */
+#define SIGRTMAX        MAX_SIGNO       /* Last real time signal */
+#define _NSIG           (MAX_SIGNO + 1) /* Biggest signal number + 1 */
+#define NSIG            _NSIG           /* _NSIG variant commonly used */
```

Comment:
are `_NSIG`, `NSIG`, `MAX_SIGNO`, `SIGSTDMAX`, `SIGSTDMIN`, etc standard names? I haven't looked yet.

Reply:
> `_NSIG`, `NSIG`

I can't find this in the standard, but pretty common in implementations to have these for calculation:

https://github.com/torvalds/linux/blob/master/include/uapi/asm-generic/signal.h#L7